src/test/java/com/transferwise/openbanking/client/api/registration/RestRegistrationClientTest.java

```diff
@@ -307,7 +309,7 @@ void updateRegistrationThrowsApiCallExceptionOnApiCallFailure() {
     void deleteRegistrationSupportsDifferentAuthenticationScopes(Set<Scope> registrationAuthenticationScopes,
                                                                  String expectedAuthenticationScope) {
 
-        AspspDetails aspspDetails = aAspspDefinition(false, registrationAuthenticationScopes);
+        AspspDetails aspspDetails = aAspspDetails(false, registrationAuthenticationScopes);
```

Comment:
can we rename it to `getTestAspspDetails` ?

Reply:
It is just a implementation details but I can rename it to `aTestAspspDetails`